### PR TITLE
Mobile support for controlling bird

### PR DESCRIPTION
**Pull request by MatCorgi**
Uses ``touches`` from p5 to check if user has touched screen
probably not that useful

### CODE REVIEW

File: sketch.js [hunk 1]
```diff
@@ -28,7 +32,7 @@ function draw() {
   background(0);
   // Draw our background image, then move it at the same speed as the pipes
   image(bgImg, bgX, 0, bgImg.width, height);
-  bgX -= pipes[0].speed;
+  bgX -= speed*0.5;
```
Comment (meiamsome) [inline sketch.js]: I would prefer this change (Parallax, using a speed variable) to be separated from the touch controls PR, please. Also you misspelt speed in your variable declaration.

**Comment by MatCorgi** [comment]
Oops i meant to change those in another branch then make it a separate pull request

**Comment by DaGuT** [comment]
dem, nice control!

**Comment by shiffman** [comment]
I like having this here! I wonder -- is there a simpler way to implement just using the p5 touch events and not worrying about multi-touch?

**Comment by DaGuT** [comment]
@shiffman  there is....
@MatCorgi please, change all your code to just new function in sketch.js 
:D

Just add
```
function touchStarted() {
   bird.up();
}
```

**Comment by MatCorgi** [comment]
I have tried that before and for some reason the bird jumps twice when touching quickly.

**Comment by Versatilus** [comment]
Awesome! Thank you!

**Pull request merged**